functions/Get-DbaQueryStoreConfig.ps1

```diff
@@ -0,0 +1,176 @@
+        $QSCDT.Columns.Add($QSCDT_MaxSize)
+        $QSCDT.Columns.Add($QSCDT_CurrentSize)
+        $QSCDT.Columns.Add($QSCDT_CaptureMode)
+        $QSCDT.Columns.Add($QSCDT_CleanupMode)
+        $QSCDT.Columns.Add($QSCDT_StaleQueryThreshold)
+	}
+
+	PROCESS
+	{
+		foreach ($servername in $SqlServer)
+		{
+            Write-Verbose "Connecting to $servername"
+			try
+			{
+				$server = Connect-SqlServer -SqlServer $servername -SqlCredential $Credential
+
+			}
+			catch
+			{
+				if ($SqlServer.count -eq 1)
+				{
+					throw $_
+				}
+				else
+				{
+					Write-Warning "Can't connect to $servername. Moving on."
+					Continue
+				}
+			}
+
+            $sqlVersion = $server.Version
+
+            $sqlVersion = $sqlVersion.ToString().Split(".")[0]
+
+            if($sqlVersion -ilt "13")
+                {
+
+                Write-Warning "The SQL Server Instance has a lower SQL Server version than SQL Server 2016. Skipping server."
+                break
+                }
+
+            
+            # We have to exclude all the system databases since they cannot have the Query Store feature enabled
+			$dbs = $server.Databases | Where-Object { $_.name -ne 'TempDb' -and $_.name -ne 'master' -and $_.name -ne 'msdb' -and $_.name -ne 'model' }
+
+			if ($databases.count -gt 0)
+			{
+				$dbs = $dbs | Where-Object { $databases -contains $_.Name }
+			}
+
+			if ($exclude.count -gt 0)
+			{
+				$dbs = $dbs | Where-Object { $exclude -notcontains $_.Name }
+			}
+
+
+			foreach ($db in $dbs)
+			{
+                $result = $null
+				Write-Verbose "Processing $($db.name) on $servername"
+
+				if ($db.IsAccessible -eq $false)
+				{
+					Write-Warning "The database $($db.name) on server $servername is not accessible. Skipping database."
+					Continue
+				}
+
+                # Start processing
+
+                $QSCDT_Row = $QSCDT.NewRow()
+
+                $QSCDT_Row.Instance = $servername
+                $QSCDT_Row.Database = $db.name
+                $QSCDT_Row.ActualState = $db.QueryStoreOptions.ActualState
+                $QSCDT_Row.FlushInterval = $db.QueryStoreOptions.DataFlushIntervalInSeconds
+                $QSCDT_Row.CollectionInterval = $db.QueryStoreOptions.StatisticsCollectionIntervalInMinutes
+                $QSCDT_Row.MaxSize = $db.QueryStoreOptions.MaxStorageSizeInMB
+                $QSCDT_Row.CurrentSize = $db.QueryStoreOptions.CurrentStorageSizeInMB
+                $QSCDT_Row.CaptureMode = $db.QueryStoreOptions.QueryCaptureMode
+                $QSCDT_Row.CleanupMode = $db.QueryStoreOptions.SizeBasedCleanupMode
+                $QSCDT_Row.StaleQueryThreshold = $db.QueryStoreOptions.StaleQueryThresholdInDays
+
+                $QSCDT.Rows.Add($QSCDT_Row)
+                       
+			}
+        }
+
+    $QSCDT | Format-Table -Property Instance, Database, ActualState, FlushInterval, CollectionInterval, MaxSize, CurrentSize, CaptureMode, CleanupMode, StaleQueryThreshold -Wrap -AutoSize 
```

Comment:
`Format-Table` should be removed, this causes issues if the output needs to be piped to another command at some point. As well, we should output objects and format commands do not do that.

You can take a look at `Get-DbaProcess` to see use of an [internal function for outputting objects](https://github.com/sqlcollaborative/dbatools/blob/master/functions/Get-DbaProcess.ps1#L184).